schema.json

```diff
@@ -150,7 +150,28 @@
           "type": "string"
         }
       },
-      "required": ["column", "down", "name", "table", "type", "up"],
+      "required": ["table", "column"],
+      "oneOf": [
+        {
+          "required": ["name"],
+          "not": {
+            "required": ["up", "down"]
+          }
+        },
+        {
+          "required": ["up", "down"],
+          "oneOf": [
+            {"required": ["check"]},
+            {"required": ["type"]},
+            {"required": ["nullable"]},
+            {"required": ["unique"]},
+            {"required": ["references"]}
+          ],
+          "not": {
+            "required": ["name"]
+          }
```

Comment:
I think this is not needed

Reply:
Removing it allows this, which isn't valid:

```json
{
  "table": "my_table",
  "column": "my_column",
  "name": "foo",
  "up": "foo",
  "down": "foo",
}
```

`up` and `down` aren't valid with `name`.